test/sortedvector.jl

```diff
@@ -21,3 +23,7 @@ using Test
         end
     end
 end;
+
+sd = TracerSparsityDetector(SortedVector{UInt, Vector{UInt}})
+@test ADTypes.jacobian_sparsity(diff, rand(10), sd) isa SparseMatrixCSC
+@test_broken ADTypes.hessian_sparsity(x -> sum(abs2, diff(x)), rand(10), sd) isa SparseMatrixCSC
```

Comment:
Why would this be broken?

Reply:
Because it requires `copy!`, see #36

Reply:
See https://github.com/adrhill/SparseConnectivityTracer.jl/issues/39